### DISCUSSION TITLE
Numpy docstring for fixed set of values

### DISCUSSION THREAD

**Discussion by blaylockbk** [mkdocstrings/mkdocstrings#617] [Q&A] [answered]
I apologize if this isn't the right place to post this (I'm just starting with mkdocstrings and am loving it! It's much easier than sphinx. Thank you).

The numpy docstring style guide says[^1]

> When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:
> 
> ```
> order : {'C', 'F', 'A'}
>    Description of `order`. 
> ```

So, when I document this...

```
Parameters
----------
order : {'C', 'F', 'A'}
    Description of `order`.
something_else : list
    Description of `something_else`.
a : int
    Description of `a`
b : str
    Description of `b`
```

...I expected mkdocstring to render the options for "order" in the "Type" column, but they are not rendered. 

![image](https://github.com/mkdocstrings/mkdocstrings/assets/6249613/d94a51f5-f267-41e7-a97e-d336bcac0d19)

If I break away from the numpy style, I can get a list, but I'd rather follow the official numpy style.
![image](https://github.com/mkdocstrings/mkdocstrings/assets/6249613/ec69a929-00c9-411b-8205-c297356839e5)

Is this a bug in mkdocstring?

[^1]: https://numpydoc.readthedocs.io/en/latest/format.html#parameters

**Comment by pawamoy** [comment]
Looks like a regression, thanks for the report :slightly_smiling_face:  Will try to fix asap

**Reply by pawamoy** [reply, marked as answer]
https://github.com/mkdocstrings/griffe/issues/212
